features/facebookImport/src/model/storage.js

```diff
@@ -37,15 +37,30 @@ export class ZipFile {
     constructor(file, pod) {
         this._pod = pod;
         this._file = file;
+
+        this._entriesSet = new Set();
     }
 
     get id() {
         return this._file.id;
     }
 
-    async getEntries() {
+    async _readEntriesList() {
         const { polyOut } = this._pod;
-        return polyOut.readdir(this.id);
+        return await polyOut.readdir(this.id);
```

Comment:
Nit: The `await` isn't necessary here because `readdir` returns a promise anyway.